### PR TITLE
Fix release 20220908 docs

### PR DESCRIPTION
**Pull request by ivarpruijn**
Fixes:
- doctests.
- Issue with `_` for variables in docs italizing content (https://github.com/objectiv/objectiv-analytics/pull/1214#event-7330974515).
- Parsing issue in series_datetime docstring.

TODO:

- [ ] Open PR with new docs in objectiv.io repo and deploy; draft here: https://github.com/objectiv/objectiv.io/pull/550
- [ ] Release new Bach/MH packages.
- [ ] Merge https://github.com/objectiv/objectiv-analytics/pull/1230 and rebuild & release Docker demo containers.

### CODE REVIEW

File: bach/docs/source/example-notebooks/open-taxonomy.rst [hunk 1]
```diff
@@ -315,8 +315,8 @@ From the marketing context, for example, we can therefore also get the 'source'
 
 When instantiating the model hub, global contexts are added using the name of the context without the word
 'Context' and converted to 'snake_case' (the name of the context split before every capital letter and
-joining them with `_`). I.e. 'HttpContext' becomes 'http' and 'InputValueContext' becomes 'input_value' when
-added as `global_context`:
+joined with an underscore), i.e. 'HttpContext' becomes 'http' and 'InputValueContext' becomes 'input_value' 
+when added to `global_contexts`:
```
Comment (thijs-obj) [inline bach/docs/source/example-notebooks/open-taxonomy.rst:319]:
```suggestion
joined with an underscore), i.e. to add the HttpContext use 'http' and to add the InputValueContext use
'input_value':
```


All renders good, but I do have a suggestion for the phrasing actually. I think the suggested change makes it clearer that this conversion to snake_case is something that the user has to do.

Comment (ivarpruijn) [reply bach/docs/source/example-notebooks/open-taxonomy.rst:319]: I like it, let's use your suggestion.

**Review by thijs-obj — COMMENTED** [review]
Left a suggestion, but OK to merge as is too.

All renders good as far as I can tell :+1:

**Comment by ivarpruijn** [comment]
Also updated the notebook here @thijs-obj : https://github.com/objectiv/objectiv-analytics/pull/1231/commits/1614e4b570b6e289dfc01a58fc862f8743e708ad - please feel free to hit the merge button if you're OK with this last change.

**Review by thijs-obj — APPROVED** [review]

**Pull request merged**